unit-9/exercise-2.js

```diff
@@ -0,0 +1,11 @@
+const ageclass = (age) => {
+	if( age < 16 ) {
+    return 'Bạn đã quá tuổi để vào lớp 10'
+	}
+	if( age === 16 ) {
+    return 'Bạn đủ tuổi vào lớp 10'
+	}
+	if( age > 16 ) {
+    return 'Bạn chưa đủ tuổi để vào lớp 10'
+	}
```

Comment:
Rảnh gì đi viết 2 lần vậy
```suggestion
if ( age >= 16 ) {
    return 'Bạn đủ tuổi vào lớp 10'
}
```